tests/utils/verification/volumeproperties.go

```diff
@@ -101,14 +102,17 @@ func GetVolumePropertiesDockerCli(volName string, hostname string) string {
 func CheckVolumeAvailability(hostName string, volumeName string) bool {
 	log.Printf("Checking volume [%s] availability from VM [%s]\n", volumeName, hostName)
 
-	volumes := GetDockerVolumes(hostName)
+	volumes, err := getDockerVolumes(hostName)
+	if err != nil {
+		return false
+	}
 	//TODO: add more detailed verification here, e.g. checking volume driver name
 	return strings.Contains(volumes, volumeName)
 }
 
-// GetDockerVolumes returns all docker volumes available from the given host
-func GetDockerVolumes(hostName string) string {
-	return ExecCmd(hostName, dockercli.ListVolumes)
+// getDockerVolumes returns all docker volumes available from the given host
```

Comment:
Pls. merge with above function, its used nowhere else and its just a call to return InvokeCommand result.